test/verify/check-machines

```diff
@@ -778,7 +778,6 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
 
             m.execute("virsh pool-define-as iscsi-pool --type iscsi --target /dev/disk/by-id --source-host 127.0.0.1 --source-dev {0} && virsh pool-start iscsi-pool".format(target_iqn))
             wait(lambda: "unit:0:0:0" in self.machine.execute("virsh pool-refresh iscsi-pool && virsh vol-list iscsi-pool"), delay=3)
-
```

Comment:
In this "tests: testMachines:tesStorageVolumesCreate remove unecessary pool cleanup" commit, can you please fix the tes*t*Storage typo? Most typos/grammar issues in the commit logs are not important, but one often greps for test names/identifiers in `git log`.